include/elections.hrl

```diff
@@ -3,7 +3,7 @@
 -record(candidate,
     {
         name :: name(),
-        party :: nil | atom()
+        party :: binary() | undefined
```

Comment:
👏 for idiomatic erlang... `nil` booooo

Reply:
Yeah, I can't remember why I had that. Probably just a brain fart.